### PR TITLE
add --include

### LINKED ISSUE #2923: include on the commandline
for debuging it would be usefull to have the opportunity to use `include` on the command line.
Example:

```
firejail --noprofile --include=disable-programs --include=disable-common bash
firejail --profile=firefox --include=$HOME/temp-firefox.local
```

### PR DESCRIPTION
**Pull request by rusty-snake**
closes #2923

### CODE REVIEW

**Comment by smitsohu** [comment]
Looks good. I still wonder if `--include` and `--profile` in the end should do the profile search the same way, including handling of file extensions.

Then the only difference would be that `--profile` overrides automatic profile loading, whereas `--include` just loads additional instructions from a file.

File: src/firejail/main.c [hunk 1]
```diff
@@ -1767,6 +1767,29 @@ int main(int argc, char **argv, char **envp) {
...
+		else if (strncmp(argv[i], "--include=", 10) == 0) {
+			char *ppath = expand_macros(argv[i] + 10);
+			if (!ppath)
+				errExit("strdup");
+
+			char *ptr = ppath;
+			while (*ptr != '/' && *ptr != '\0')
+				ptr++;
+			if (*ptr == '\0') {
+				// ppath contains no '/', assume it's a name
```
Comment (smitsohu) [inline src/firejail/main.c:1779]: Would it be possible to somehow prioritize search in the current working directory?

Comment (rusty-snake) [reply src/firejail/main.c]: Done.

> Looks good to me. I still wonder if --include and --profile in the end should do the profile search the same way, including handling of file extensions.
>
>Then the only difference would be that --profile overrides automatic profile loading, whereas --include just loads additional instructions from a file.

I tried first, but I don't completely understand the `--profile` code. What does `has_colon`.

**Comment by netblue30** [comment]
@rusty-snake -  looks fine, are you merging it?

**Pull request merged**